api/charmrevisionupdater/updater_test.go

```diff
@@ -4,57 +4,32 @@
 package charmrevisionupdater_test
 
 import (
-	"github.com/juju/charm/v7"
-	jc "github.com/juju/testing/checkers"
-	"github.com/juju/utils"
 	gc "gopkg.in/check.v1"
 
+	basetesting "github.com/juju/juju/api/base/testing"
 	"github.com/juju/juju/api/charmrevisionupdater"
-	"github.com/juju/juju/apiserver/facades/controller/charmrevisionupdater/testing"
-	jujutesting "github.com/juju/juju/juju/testing"
-	"github.com/juju/juju/state"
+	"github.com/juju/juju/apiserver/params"
 )
 
-type versionUpdaterSuite struct {
-	jujutesting.JujuConnSuite
-	testing.CharmSuite
-
-	updater *charmrevisionupdater.State
-}
+type versionUpdaterSuite struct{}
```

Comment:
it is best practice to use either testing.IsolationSuite or BaseSuite to capture logging

Reply:
ah i did it for every other one, missed this one sorry